### PR TITLE
Split out Stanford SASS for Instructor Dashboard

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

**Comment by caesar2164** [comment]
should the files have "_stanford" in the filename so that they'll never have any conflict?

File: lms/static/sass/course/instructor/_email_query.scss [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  .queries-section {
+    margin-bottom: ($baseline*1.5);
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+  .email-lists-management {
+    .wrapper-email-select {
+      background-color: $light-gray;
+      padding: ($baseline/2);
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: this *should* go first

File: lms/static/sass/course/instructor/_email_query.scss [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+          color: $lighter-base-font-color;
+          line-height: 1.3em;
+          font-size: .85em;
+        }
+      }
+    }
+  }
+  .email-list {
+    @include box-sizing(border-box);
+    background-color: $light-gray;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: should go after padding

File: lms/static/sass/course/instructor/_email_query.scss [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+    }
+    .active {
+      display: table-cell;
+    }
+    .addingQuery {
+      text-align: right;
+    }
+    .addQuery {
+      @include idashbutton($blue);
+      font-size: 12px;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: after "right"

File: lms/static/sass/course/instructor/_email_query.scss [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+    .loadQuery {
+      color: $black;
+      cursor: pointer;
+      text-align: center;
+      &:hover, &:focus {
+        color: $blue;
+      }
+    }
+    .tableCaption {
+      text-align: left;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: after font-weight

File: lms/static/sass/course/instructor/_email_query.scss [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+      font-weight: bold;
+    }
+    thead {
+      color:white;
+    }
+    td {
+      @extend %t-copy-sub1;
+      vertical-align: middle;
+      word-wrap: break-word;
+      padding: ($baseline/2) ($baseline/4);
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: before vertical-align

File: lms/static/sass/course/instructor/_email_query.scss [hunk 6]
```diff
@@ -0,0 +1,204 @@
...
+      border-left: 1px solid $light-gray;
+      border-right: 1px solid $light-gray;
+    }
+    .estimated {
+      color: #1E90FF;
+      font-size: 20px;
+      padding-bottom: $baseline / 2;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: should go first

File: lms/static/sass/course/instructor/_email_query.scss [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+      position: relative;
+      padding: $baseline;
+      margin-top: -1px;
+      border: 1px solid $light-gray;
+      background-color: #efefef;
+      box-shadow: inset #bbb 0 1px 1px 0;
+    }
+    .savequeryname {
+      float: right;
+      right: $baseline;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: first

File: lms/static/sass/course/instructor/_email_query.scss [hunk 8]
```diff
@@ -0,0 +1,204 @@
...
+      font-size: 15px;
+      margin-right: 5px;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_email_query.scss]: second

File: lms/static/sass/course/instructor/_graph_forums.scss [hunk 9]
```diff
@@ -0,0 +1,35 @@
+.instructor-dashboard-wrapper-2 section.idash-section#data_download {
+  .reports-download-container {
+    .graph-forums {
+      cursor: pointer;
+      float:right;
+      padding-left: 25px;
+      padding-right: 10px;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_graph_forums.scss]: paddings should go first

File: lms/static/sass/course/instructor/_lti_upload.scss [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+      top: 0;
+      left: 0;
+      z-index: 1;
+    }
+    .fileupload {
+      position: relative;
+      z-index: 2;
+      opacity: 0;
+      height: ($baseline*1.5);
+      width: ($baseline*6);
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_lti_upload.scss]: height and width before z-index and opacity

File: lms/static/sass/course/instructor/_lti_upload.scss [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    }
+  }
+  .bordered-fieldset {
+    border: solid 4px #000000;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_lti_upload.scss]: border after margin and padding

File: lms/static/sass/course/instructor/_lti_upload.scss [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+  }
+  .select-tag {
+    width: 40%;
+    height: ($baseline*1.5);
+    font-size: 1.1em;
+  }
+  .output-box {
+    .loading {
+      font-size: 2em;
+      margin-bottom: $baseline;
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_lti_upload.scss]: margins go first

**Comment by stvstnfrd** [comment]
@caesar2164 Ideally, I'd like to split these files out into the `openedx/stanford` hierarchy, but for now, I didn't want to go down the rabbit hole of making sure `collect_static_assets` finds the new paths.

**Comment by stvstnfrd** [comment]
@caesar2164 Sorry, I didn't exactly address your comment; yeah, I guess I can move those into a `stanford/` subdirectory.

File: lms/static/sass/course/instructor/_instructor_2.scss [hunk 13]
```diff
@@ -2518,66 +2212,7 @@ input[name="subject"] {
...
-    .loading {
-      font-size: 2em;
-      margin-bottom: $baseline;
-    }
-  }
-}
+@import "course/instructor/delete_report";
+@import "course/instructor/email_query";
+@import "course/instructor/graph_forums";
+@import "course/instructor/lti_upload";
```
Comment (caesar2164) [inline lms/static/sass/course/instructor/_instructor_2.scss]:
```
@import "stanford/course/instructor/delete_report";
@import "stanford/course/instructor/email_query";
@import "stanford/course/instructor/lti_upload";
@import "stanford/course/instructor/lti_upload";
```

**Pull request merged**